ArcGISRuntimeSDKQt_CppSamples/Layers/RasterFunctionFile/RasterFunctionFile.cpp

```diff
@@ -89,14 +93,15 @@ RasterFunction* RasterFunctionFile::createRasterFunction()
   // create a RasterFunction
   RasterFunction* rasterFunction = new RasterFunction(m_dataPath + "/color.json");
 
+  // check for valid raster function
+  if (!rasterFunction)
```

Comment:
Will it ever fail this check? If the path is incorrect, the new will still pass. Ok to keep it like this but its not going to satisfy this condition if the file doesn't exist.